src/python/grpcio_test/grpc_test/_adapter/_low_test.py

```diff
@@ -193,13 +193,13 @@ def testEcho(self):
       self.assertNotIn(client_result.type, found_client_op_types)
       found_client_op_types.add(client_result.type)
       if client_result.type == _types.OpType.RECV_INITIAL_METADATA:
-        self.assertEqual(dict(server_initial_metadata),
-                         dict(client_result.initial_metadata))
+        self.assertLessEqual(set(server_initial_metadata),
```

Comment:
Let's reduce the number of definitions of metadata transmission that we have running around the codebase and delegate this judgement to grpc_test.test_common.metadata_transmitted?